install-dev/init.php

```diff
@@ -92,6 +85,24 @@
     $kernel->loadClassCache();
     $kernel->boot();
 }
+
+if (!defined('_THEME_NAME_')) {
+    if (getenv('_PS_THEME_NAME_') !== false) {
```

Comment:
Seems to be always defined, isn't it?

Reply:
it's always defined yes, but you can check for empty instead.

Reply:
> Seems to be always defined, isn't it?

The condition was here before so I guess it's not

Reply:
No it wasn't. I'm talking about `getenv('_PS_THEME_NAME_')`, not `!defined('_THEME_NAME_')`.

Reply:
With `    env(_PS_THEME_NAME_): "classic"` in config.yml, we will always have a value in there.

Reply:
oh yeah, make sense! :D